### PR TITLE
Feat notes add edit movetotrash

### PR DESCRIPTION
**Pull request by aishwarya-bm**
Added functionality to add/edit/delete notes

### CODE REVIEW

**Review by sadathshariff — APPROVED** [review]
Good Naming convention and The Code looks clean and understandable

File: src/components/addNote/Addnote.jsx [hunk 1]
```diff
@@ -1,21 +1,74 @@
...
+        style={{ backgroundColor: note_editor.cardColor }}
+      >
         <div className="d-flex notes-details">
           <label>
             <b>Title:</b> &nbsp;
-            <input type="text" />
+            <input
+              type="text"
+              name="title"
+              value={note_editor.title}
```
Comment (shubhamkumar648) [inline src/components/addNote/Addnote.jsx]: suggestion: you can keep  this value in destructuring  and  use only the value name for better readability

**Review by shubhamkumar648 — APPROVED** [review]
the remaining functionality is good, i like the way  you use toggle in functionality . good to go

File: src/components/addNote/Addnote.jsx (hunk 1 above)
Comment (aishwarya-bm) [reply src/components/addNote/Addnote.jsx]: Oh yes will make that change,thanks!

**Pull request merged**